aws/aws-event-streams/src/main/java/software/amazon/smithy/java/aws/events/RpcEventStreamsUtil.java

```diff
@@ -31,7 +31,7 @@ public static Flow.Publisher<ByteBuffer> bodyForEventStreaming(
         Flow.Publisher<SerializableStruct> eventStream = input.getMemberValue(streamingMember(input.schema()));
         var publisher = EventStreamFrameEncodingProcessor.create(eventStream, eventStreamEncodingFactory);
         // Queue the input as the initial-request.
-        publisher.onNext(input);
+        publisher.enqueueItem(input);
```

Comment:
Presumably, you are doing it this way because we may not have been subscribed to yet. If so, our reactive code is broken elsewhere (maybe here?) and must be brought back into compliance with the reactive streaming specification.